src/routes/products/messaging/+page.svelte

```diff
@@ -88,24 +89,11 @@ try await messaging.createSubscriber(
 
     const codeMessage = [
         {
-            language: 'js',
+            language: "server-nodejs",
             platform: 'Node.js',
             content: `import { Messaging, ID } from "node-appwrite"
 
 const messaging = Messaging(client);
-await messaging.createPush(
-    ID.unique(),           // Message ID
-    'Breaking update',     // Push title
-    'Hello, world!',       // Push body
-    ['news', 'sport'],       // Topic IDs
-);`
-        },
-        {
-            language: 'ts',
-            platform: 'Deno',
-            content: `import { Messaging, ID } from "https://deno.land/x/appwrite/mod.ts";
-
-const messaging = new Messaging(client);
```

Comment:
I think you accidentally removed deno 👍🏻

Reply:
Naa this was intentional.
The task mentioned to use only `Node.js`, so I removed this. Need to add back?